e2e/utils.go

```diff
@@ -536,12 +571,22 @@ func validatePVCClone(sourcePvcPath, clonePvcPath, clonePvcAppPath string, f *fr
 		e2elog.Failf("failed to load application with error %v", err)
 	}
 	appClone.Namespace = f.UniqueName
+	appClone.Labels = label
 	wg.Add(totalCount)
 	// create clone and bind it to an app
 	for i := 0; i < totalCount; i++ {
 		go func(w *sync.WaitGroup, n int, p v1.PersistentVolumeClaim, a v1.Pod) {
 			name := fmt.Sprintf("%s%d", f.UniqueName, n)
 			wgErrs[n] = createPVCAndApp(name, f, &p, &a, deployTimeout)
+			if *pvc.Spec.VolumeMode == v1.PersistentVolumeFilesystem {
+				filePath := appClone.Spec.Containers[0].VolumeMounts[0].MountPath + "/test"
+				e2elog.Logf("Calculating checksum")
```

Comment:
nothing much useful with this log. can you remove it or log error and checkSumClone vault at line 586